cmd/ipfs-cluster-service/main.go

```diff
@@ -289,6 +298,25 @@ remove the %s file first and clean any Raft state.
 					cfgs.clusterCfg.Secret = userSecret
 				}
 
+				peersOpt := c.String("peers")
+				multiAddrs := []ma.Multiaddr{}
+				if peersOpt != "" {
+					addrs := strings.Split(peersOpt, ",")
```

Comment:
there is an if inside an if here, but you could do ` addrs := strings.Split(c.String("peers"), " ,")`  directly and have a single `if len(addrs) > 0` .

Reply:
In fact you don't even need an `if len(addrs) > 0` at that point since the loop won't even run with 0 elements. You can put an if `len(peers) > 0` below and do the trusted_peer, init_peerset assignments and peerstore writing inside it.

Reply:
but I can still remove that if condition

Reply:
true